internal/config/report.go

```diff
@@ -90,3 +117,56 @@ func (c *ReportConfig) splitSeverity(logger *zap.SugaredLogger, severity string)
 	}
 	return severities
 }
+
+// mapFormat maps format and output for multi-format report
+func (c *ReportConfig) mapOutput(format []string, logger *zap.SugaredLogger) (output []*os.File, err error) {
+	for _, v := range strings.Split(c.output, ",") {
+		var out *os.File
+		if v != "" && v != "stdout" {
+			if out, err = os.Create(v); err != nil {
+				return output, xerrors.Errorf("failed to create an output file: %w", err)
+			}
+		} else {
+			out = os.Stdout
+		}
+		output = append(output, out)
+	}
+
+	for i := len(output); i < len(format); i++ {
+		output = append(output, os.Stdout)
+	}
+
+	return output, nil
+}
+
+// mapFormat maps format and template for multi-format report
+func (c *ReportConfig) mapTemplate(format []string, logger *zap.SugaredLogger) (template []string) {
+	var isTemplate bool
+	formats := make(map[string]struct{})
+
+	for _, f := range format {
+		_, ok := formats[f]
+		if ok {
+			template = append(template, "")
+			logger.Warnf("--format %s is ignored because it has been specified.", f)
+			continue
+		}
+		formats[f] = struct{}{}
```

Comment:
I can remove these lines if we can assume that format will not be repeated e.g no `--format json,template,template`